goas/zone/risc0_proofs/spend_zone_funds/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "spend-zone-funds"
```

Comment:
are dashes kosher in package names?

Reply:
I think unclear https://rust-lang.github.io/api-guidelines/naming.html, there are high profile crates using them (e.g. tokio) but we can decide to stick to one

Reply:
Looks like it doesn't matter? can just leave it as is